kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -103,6 +109,12 @@ type Reconciler struct {
 	metricsConfig  *metrics.ExporterOptions
 
 	sinkResolver *resolver.URIResolver
+
+	minReplicaCount     *int32
+	maxReplicaCount     *int32
+	cooldownPeriod      *int32
+	pollingInterval     *int32
+	triggerLagThreshold *int32
```

Comment:
`lagThreshold` in current keda master is an int64

Reply:
changed